e2e/go.mod

```diff
@@ -28,6 +28,7 @@ require (
 
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/internal v1.10.0 // indirect
+	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerregistry/armcontainerregistry v1.3.0-beta.1 // indirect
```

Comment:
Had to use 1.3.0-beta.1 in order to have access to AnonymousPullEnabled in ACR creation